Installation fails on Bash on Ubuntu on Windows 10

Installation fails with 

```
$ make install
# Copying .bash_prompt to ~/.bash_prompt
cp -f ".bash_prompt" "/home/chris/.bash_prompt"
./install.bash
./install.bash: line 2: $'\r': command not found
./install.bash: line 4: syntax error near unexpected token `$'{\r''
'/install.bash: line 4: `ps1_is_installed () {
make: *** [install] Error 2
```

This is a line-ending issue.
Doing ```sed -i 's/\r$//' sed -i 's/\r$//' install.bash``` before make install
gets one step further, but then

```
$ make install
# Copying .bash_prompt to ~/.bash_prompt
cp -f ".bash_prompt" "/home/chris/.bash_prompt"
./install.bash
: command not found
bash: /home/chris/.bash_prompt: line 100: syntax error near unexpected token `$'{\r''
'ash: /home/chris/.bash_prompt: line 100: `function sexy_bash_prompt_get_git_branch() {
# Adding ~/.bash_prompt to ~/.bashrc
: command not found
bash: /home/chris/.bash_prompt: line 100: syntax error near unexpected token `$'{\r''
'ash: /home/chris/.bash_prompt: line 100: `function sexy_bash_prompt_get_git_branch() {
: command not found
bash: /home/chris/.bash_prompt: line 100: syntax error near unexpected token `$'{\r''
'ash: /home/chris/.bash_prompt: line 100: `function sexy_bash_prompt_get_git_branch() {
# We cannot confirm that sexy-bash-prompt has installed properly
# Please open a new terminal window to confirm

# If it has not, please run the following code:
echo ". ~/.bashrc" >> "/home/chris/.profile"
make: *** [install] Error 1
```

So another ```sed -i 's/\r$//' ~/.bash_prompt``` to resolve this.

It did seem to update the .bashrc correctly (added this at the end)
```
# Run twolfson/sexy-bash-prompt
. ~/.bash_prompt
```

however, mine was added multiple times (each times make install failed I guess).

So, this workaround is fine, but it would be nice to figure out how to fix this properly. I'm a bit as to where in the process invalid line breaks get introduced.

--- a/README.md
+++ b/README.md
@@ -22,7 +22,7 @@ Forked from [a gist by gf3][sexy-bash-orig].
 One line install:
 
 ```bash
-(cd /tmp && git clone --depth 1 https://github.com/twolfson/sexy-bash-prompt && cd sexy-bash-prompt && make install) && source ~/.bashrc
+(cd /tmp && git clone --depth 1 --config core.autocrlf=false https://github.com/twolfson/sexy-bash-prompt && cd sexy-bash-prompt && make install) && source ~/.bashrc
 ```
 
 ### My colors don't look as advertised
@@ -50,7 +50,7 @@ rm /tmp/.bash_prompt_term
 ### Manual install
 ```bash
 # Clone the repository
-git clone --depth 1 https://github.com/twolfson/sexy-bash-prompt
+git clone --depth 1 --config core.autocrlf=false https://github.com/twolfson/sexy-bash-prompt
 # Cloning into 'sexy-bash-prompt'...
 # ...
 # Resolving deltas: 100% (13/13), done.

